kfdefs/overlays/osc/osc-cl1/superset/externalsecrets/superset.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: external-secrets.io/v1beta1
```

Comment:
Should this fine not be named `superset-custom.yaml` to keep the filename and secret names the same?

Reply:
yep yep tyty